manpages.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+)
+
+var manpagesMatcher = regexp.MustCompile(`\b(\w+)\((\d[\da-z_-]*)\)(\W|$)`)
```

Comment:
Doesn't have to be an explicit explanation, just something like "shall match strings like "…" and extract the following parts"